addons/web_unsplash/static/src/xml/unsplash_image_widget.xml

```diff
@@ -62,6 +62,11 @@
                 </t>
                 <t t-set="showUnsplashSetup" t-value="true"/>
             </t>
+            <t t-elif="status == 'no_access'">
+                <t t-set="title">
+                    Unsplash is not set
+                </t>
+            </t>
```

Comment:
You can remove that part, we don't want the portal/end user to know about DB misconfiguration.
It should just tell him that there is no image found, that's it.
We don't want the portal user to be notified that the key is missing, that the maximum request have been reached, that the key is unauthorized etc.
Anything related to the credential should just be hidden from that user.
Just check at higher level in the templates if you can add a groups/t-if for portal user to not show that whole credential part


![image](https://user-images.githubusercontent.com/30048408/163382281-591fc786-5ad5-455b-b4d8-22ffca9ed1db.png)